lighthouse-core/scripts/release/test.sh

```diff
@@ -35,8 +35,9 @@ mkdir -p /tmp/lighthouse-local-test
 cd /tmp/lighthouse-local-test
 
 npm init -y
-npm install "$LH_PRISTINE_ROOT/lighthouse-$VERSION.tgz"
-cd node_modules/lighthouse/lighthouse-cli/test/ && npm install lodash.clonedeep && cd ../../../../
```

Comment:
this command was accidentally installing all our devDeps

idk why I had it cd to that particular folder 🤣